.gitignore

```diff
@@ -22,6 +22,7 @@
 _gh_pages
 _site
 .ruby-version
```

Comment:
Think you could just remove this line unless there are `.ruby-version`s below the project root that we want to keep ignoring

Reply:
```suggestion

```